hw/dv/data/common_sim_cfg.hjson

```diff
@@ -137,10 +140,18 @@
   tool_srcs:  ["{dv_root}/tools/waves.tcl"],
 
   // Project defaults for VCS
-  vcs_cov_hier: "-cm_hier {tool_srcs_dir}/cover.cfg"
+  vcs_cov_hier: "{{build_mode}_vcs_cov_hier}"
   vcs_cov_excl_files: ["{tool_srcs_dir}/common_cov_excl.el"]
 
+  // Build-specific coverage cfg files for VCS.
+  default_vcs_cov_hier: "-cm_hier {tool_srcs_dir}/cover.cfg"
+  cover_reg_top_vcs_cov_hier: "-cm_hier {tool_srcs_dir}/cover_reg_top.cfg"
+
   // Project defaults for Xcelium
-  // xcelium_cov_cfg_file: "-covfile {tool_srcs_dir}/cover.ccf"
-  // xcelium_cov_refine_file: ["{tool_srcs_dir}/common_cov.vRefine"]
+  // xcelium_cov_cfg_file: "{{build_mode}_xcelium_cov_cfg_file}"
+  // xcelium_cov_refine_files: ["{tool_srcs_dir}/common_cov.vRefine"]
+
+  // Build-specific coverage cfg files for Xcelium.
+  // default_xcelium_cov_cfg_file: "-covfile {tool_srcs_dir}/cover.ccf"
```

Comment:
@rasmus-madsen @senelson7 @tunghoang290780 
We need to supply the Xcelium equivalents of the above.

Reply:
@sriyerg I can add those in a later PR
it falls  nicely into the work I am doing on updating the cover methodology with examples etc.